common/src/main/java/org/jboss/hal/testsuite/finder/FinderNavigation.java

```diff
@@ -202,6 +204,10 @@ public FinderNavigation resetNavigation() {
         return this;
     }
 
+    public void clearNavigation() {
```

Comment:
Please add javadoc for this method to emphasize the difference compared to `resetNavigation` method.